{
  "name": "assert-cb",
  "version": "0.1.0",
  "description": "Assert that calls a function with an error instead of throwing it if the assert fails.",
  "main": "index.js",
  "scripts": {
    "test-main": "tap test/*.js",
    "test-0.8": "nave use 0.8 npm run test-main",
    "test-0.10": "nave use 0.10 npm run test-main",
    "test-all": "npm run test-main && npm run test-0.8 && npm run test-0.10",
    "test": "if [ -e $TRAVIS ]; then npm run test-all; else npm run test-main; fi"
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/thlorenz/assert-cb.git"
  },
  "homepage": "https://github.com/thlorenz/assert-cb",
  "dependencies": {},
  "devDependencies": {
    "nave": "~0.4.3",
    "tap": "~0.4.3"
  },
  "keywords": [],
  "author": {
    "name": "Thorsten Lorenz",
    "email": "thlorenz@gmx.de",
    "url": "http://thlorenz.com"
  },
  "license": {
    "type": "MIT",
    "url": "https://github.com/thlorenz/assert-cb/blob/master/LICENSE"
  },
  "engine": {
    "node": ">=0.8"
  }
}
